src/uniparc/components/entry/XRefsFacets.tsx

```diff
@@ -1,123 +1,155 @@
 import { useMemo, FC, ReactNode } from 'react';
 import { countBy, uniqBy } from 'lodash-es';
-import { Facets, Facet } from 'franklin-sites';
+import { Facets, Loader } from 'franklin-sites';
 
 import { EntryTypeIcon } from '../../../shared/components/entry/EntryTypeIcon';
 
 import { EntryType } from '../../../uniprotkb/adapters/uniProtkbConverter';
 
 import {
-  UniParcXRef,
   databaseToEntryType,
+  UniParcAPIModel,
+  UniParcXRef,
+  XRefsInternalDatabases,
 } from '../../adapters/uniParcConverter';
-import { FacetValue } from '../../../uniprotkb/types/responseTypes';
+import {
+  FacetObject,
+  FacetValue,
+} from '../../../uniprotkb/types/responseTypes';
 import { OrganismData } from '../../../uniprotkb/adapters/namesAndTaxonomyConverter';
+import { UseDataAPIWithStaleState } from '../../../shared/hooks/useDataApiWithStale';
+
+import '../../../shared/components/results/styles/results-view.scss';
 
-const sortByCount = (a: FacetValue, b: FacetValue) => b.count - a.count;
+const reviewed: XRefsInternalDatabases = 'UniProtKB/Swiss-Prot';
```

Comment:
Having to rewrite a string seems a downside of not using enums. I know this is way is preferable as it makes it easier to use mock values. Could there be a way to have both, for example:

```
export enum XRefsInternalDatabasesEnum {
  REVIEWED = 'UniProtKB/Swiss-Prot',
  UNREVIEWED = 'UniProtKB/TrEMBL',
}
export type XRefsInternalDatabases = `${XRefsInternalDatabasesEnum}`;
```
will allow us to only write out the string once in the enum definition and we can also export the union type for the API mocks. We could then for example do:

```
const sortByCount = (a: FacetValue, b: FacetValue) => {
  if (
    a.value in XRefsInternalDatabasesEnum &&
    b.value in XRefsInternalDatabasesEnum
  ) {
    return a.value === XRefsInternalDatabasesEnum.REVIEWED ? -1 : 1;
  }
  if (a.value in XRefsInternalDatabasesEnum) {
    return -1;
  }
  if (b.value in XRefsInternalDatabasesEnum) {
    return 1;
  }
  return b.count - a.count;
};
```

Reply:
I like this approach, I changed it partially like you suggested with the enum and the stringified version of it for the type. Not really sure about the `in` test with enums though so I did keep the Set